plugins/chat/spec/system/page_objects/chat/chat.rb

```diff
@@ -25,9 +25,9 @@ def open
         visit("/chat")
       end
 
-      def open_new_message
+      def open_new_message(check: true)
```

Comment:
`check` is pretty generic...maybe `modal_should_open: true`. Just saying, it's not immediately obvious what `check` means

Reply:
:+1: went with `ensure_open`